addon-test-support/@ember/test-helpers/dom/double-click.ts

```diff
@@ -80,6 +82,9 @@ export function __doubleClick__(element: Element | Document, options: MouseEvent
   doubleClick('button', { shiftKey: true });
 */
 export default function doubleClick(target: Target, options: MouseEventInit = {}): Promise<void> {
```

Comment:
Lets call this `_options` and not initialize it

```suggestion
export default function doubleClick(target: Target, _options: MouseEventInit): Promise<void> {
```